editor/debugger/script_editor_debugger.cpp

```diff
@@ -2184,11 +2195,32 @@ ScriptEditorDebugger::ScriptEditorDebugger() {
 	{ //vmem inspect
 		VBoxContainer *vmem_vb = memnew(VBoxContainer);
 		HBoxContainer *vmem_hb = memnew(HBoxContainer);
-		Label *vmlb = memnew(Label(TTR("List of Video Memory Usage by Resource:") + " "));
-		vmlb->set_theme_type_variation("HeaderSmall");
 
-		vmlb->set_h_size_flags(SIZE_EXPAND_FILL);
+		Label *vmlb = memnew(Label(TTR("List of Video Memory Usage by Resource:")));
```

Comment:
```suggestion
		Label *vmlb = memnew(Label(TTRC("List of Video Memory Usage by Resource:")));
```

@KoBeWi Is this correct?

Reply:
Yes, the tooltip should also use TTRC.